mysql-test/suite/tianmu/t/issue763.test

```diff
@@ -0,0 +1,73 @@
+--source include/have_tianmu.inc
+
+--echo #
+--echo # SELECT ... FROM tablename WHERE col LIKE NULL
+--echo #
+
+--disable_warnings
+DROP DATABASE IF EXISTS test_like_null;
+--enable_warnings
+
+CREATE DATABASE test_like_null;
+USE test_like_null;
+
+create table tlike1 (id int, val varchar(255));
+INSERT INTO tlike1 VALUES(1,'a'), (1,'b'), (1,'c'), (2,'dd'), (3,'eee'), (4, NULL), (5, NULL);
+SELECT * FROM tlike1;
+SELECT * FROM tlike1 WHERE val is NULL;
+SELECT * FROM tlike1 WHERE val LIKE NULL;
+SELECT COUNT(*) FROM tlike1 WHERE val LIKE NULL;
+SELECT id, val FROM tlike1 WHERE id=1 and val LIKE NULL;
+SELECT id, val FROM tlike1 WHERE id=1 or val LIKE NULL;
+SELECT id, val FROM tlike1 WHERE id in (4, 5) and val LIKE NULL;
+SELECT id, val FROM tlike1 WHERE id in (4, 5) or val LIKE NULL;
```

Comment:
Would you like to add some cases for negtive condition, such as `not like`.

Reply:
I'll make the requested changes!